os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -464,6 +464,8 @@ void up_assert(const uint8_t *filename, int lineno)
 
 	up_dumpstate();
 
+	elf_show_all_bin_addr();
```

Comment:
Could you check the build break?
You can see the CI build log.
Maybe it is related to config.

Reply:
Rectified